src/lib/client.js

```diff
@@ -185,6 +202,18 @@ class Client extends EventEmitter {
 
     return this.plugin.send(transfer)
   }
+
+  /**
+   * Get the list of connector URIs.
+   * @returns {Promise.<URI[]>}
+   */
+  getConnectors () {
```

Comment:
Ok, https://github.com/interledger/js-ilp-plugin-bells/pull/42 caches the return value of `getInfo`, which includes the connectors.